src/brevitas_examples/llm/main.py

```diff
@@ -311,11 +312,14 @@ def quantize_llm(args, extra_args=None):
     if args.magr and not args.load_checkpoint:
         print("Applying MagR...")
         model = offload_model(model)
+        # save original weights for GPFQ or Qronos since they both require optimizing
+        # a mismatched objective function (see https://arxiv.org/abs/2505.11695), or
+        # if explicitly specified in the CLI
+        _create_weight_orig = args.gpfq or args.qronos or args.gpxq_create_weight_orig
```

Comment:
I don't like this.
The user is fully responsible to set the flag if they are using GPFQ or Qronos.

We should just document somewhere that they are needed (e.g., in the yamls to reproduce the results)